File: Word-Formed/src/com/google/corrigan/owen/wordformed/GameOverActivity.java
package com.google.corrigan.owen.wordformed;

import java.util.List;

import android.app.Activity;
import android.app.ListActivity;
import android.os.Bundle;
import android.util.Log;
import android.widget.ArrayAdapter;
import android.widget.TextView;

public class GameOverActivity extends ListActivity
{
	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.game_over);
		TextView t=(TextView)findViewById(R.id.score_result);
		t.setText("Congratulations, you acheived a score of " + SinglePlayerGameView.getScore());
		
		List<Word> values = SinglePlayerGameView.getWords();

		// Use the SimpleCursorAdapter to show the
		// elements in a ListView
		WordAdapter adapter = new WordAdapter(this, values);
		setListAdapter(adapter);
	}
}

File: app/src/main/java/com/google/corrigan/owen/wordformed/GameOverActivity.kt
package com.google.corrigan.owen.wordformed

import android.os.Bundle
import android.view.WindowManager
import android.widget.ListView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class GameOverActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.game_over)

        val scoreResult = findViewById<TextView>(R.id.score_result)
        scoreResult.text = "Congratulations, you achieved a score of ${SinglePlayerGameView.score}"

        window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN
        )

        val values = SinglePlayerGameView.wordsByScore
        val listView = findViewById<ListView>(android.R.id.list)
        val adapter = WordAdapter(this, values)
        listView.adapter = adapter
    }
}
